indico/modules/rb/controllers/backend/bookings.py

```diff
@@ -187,6 +189,7 @@ def _process(self):
         except NoReportError as e:
             db.session.rollback()
             raise ExpectedError(str(e))
+        self.after_create(args, reservation=resv)
```

Comment:
Can't you use the existing `booking_created` signal for this?

Reply:
At the current state, the signal does not contain the "extra fields".
I've just tested this and [here](https://github.com/indico/indico/blob/c87a07e1d68082364c1427e4b377ca485cd05547/indico/modules/rb/models/reservations.py#L382) `data` variable contains it.
I could pass it as an argument to the signal if this approach is preferable.

Reply:
Yes, I think that's better.